test/doc_logs_bang_test.rb

```diff
@@ -0,0 +1,28 @@
+require File.expand_path(File.dirname(__FILE__) + "/test_helper")
+
+class DocLogsBangTest < MiniTest::Test
+  describe "doc_logs!" do
+    before do
+      DocRaptor.api_key "something something"
+    end
+
+    describe "with bogus arguments" do
+      it "should raise an error if something other than an options hash is passed in" do
+        assert_raises(ArgumentError) {DocRaptor.doc_logs!(true)}
+        assert_raises(ArgumentError) {DocRaptor.doc_logs!(nil)}
+      end
+    end
+
+    describe "with good arguments" do
+      it "should give me a valid response" do
+        stub_http_response_with("simple_list_docs.xml", :get)
```

Comment:
I don't believe the return is any different but I can confirm this with Tyler at DocRaptor later today.

Sent from my iPhone

On Aug 5, 2015, at 10:42 AM, janxious <notifications@github.com<mailto:notifications@github.com>> wrote:


In test/doc_logs_bang_test.rb<https://github.com/expectedbehavior/doc_raptor_gem/pull/14#discussion_r36309152>:

> +class DocLogsBangTest < MiniTest::Test
> +  describe "doc_logs!" do
> +    before do
> +      DocRaptor.api_key "something something"
> +    end
> +
> +    describe "with bogus arguments" do
> +      it "should raise an error if something other than an options hash is passed in" do
> +        assert_raises(ArgumentError) {DocRaptor.doc_logs!(true)}
> +        assert_raises(ArgumentError) {DocRaptor.doc_logs!(nil)}
> +      end
> +    end
> +
> +    describe "with good arguments" do
> +      it "should give me a valid response" do
> +        stub_http_response_with("simple_list_docs.xml", :get)


probably need to add a new fixture that is like 'simple_list_doc_logs.xml' for this case.

—
Reply to this email directly or view it on GitHub<https://github.com/expectedbehavior/doc_raptor_gem/pull/14/files#r36309152>.